turbinia/turbiniactl.py

```diff
@@ -31,6 +31,7 @@
 from turbinia.lib import libcloudforensics
 from turbinia import __version__
 from turbinia.processors import archive
+from turbinia import notify
```

Comment:
Sorry I didn't catch this last time, but can we actually move this down to around line 402?  See the comment above that block for more details, but basically anything that loads the config we want to import after the config is set up.